pkg/liqoctl/status/peer/peerinfo.go

```diff
@@ -0,0 +1,277 @@
+		if foreignCluster.Spec.ForeignProxyURL != "" {
+			authSection.AddEntry("Auth Proxy URL", foreignCluster.Spec.ForeignProxyURL)
+		}
+	}
+}
+
+// addNetworkSection adds a section about the network configuration.
+func (pic *PeerInfoChecker) addNetworkSection(ctx context.Context, rootSection output.Section,
+	foreignCluster *discoveryv1alpha1.ForeignCluster, tunnelEndpoint *netv1alpha1.TunnelEndpoint, localClusterName string) error {
+	networkSection := rootSection.AddSection("Network")
+	networkStatus := peeringconditionsutils.GetStatus(foreignCluster, discoveryv1alpha1.NetworkStatusCondition)
+	networkSection.AddEntry("Networking", string(networkStatus))
+
+	if pic.options.Verbose {
+		networkConfigs, err := getters.GetNetworkConfigsByLabel(ctx, pic.options.CRClient,
+			foreignCluster.Status.TenantNamespace.Local, labels.NewSelector())
+		if err != nil {
+			return err
+		}
+
+		localNetworkConfigSection := networkSection.AddSection("Local NetworkConfig")
+		remoteNetworkConfigSection := networkSection.AddSection("Remote NetworkConfig")
+		var selectedSection output.Section
+		var remoteSectionMsg, remotePodCIDRMsg, remoteExternalCIDRMsg string
+		for i := range networkConfigs.Items {
+			nc := &networkConfigs.Items[i]
+			if liqonetutils.IsLocalNetworkConfig(nc) {
+				selectedSection = localNetworkConfigSection
+				remoteSectionMsg = fmt.Sprintf("how %q has been remapped by %q", localClusterName, foreignCluster.Name)
+			} else {
+				selectedSection = remoteNetworkConfigSection
+				remoteSectionMsg = fmt.Sprintf("how %q remapped %q", localClusterName, foreignCluster.Name)
+			}
+
+			if nc.Status.PodCIDRNAT == consts.DefaultCIDRValue {
+				remotePodCIDRMsg = NotRemappedMsg
+			} else {
+				remotePodCIDRMsg = nc.Status.PodCIDRNAT
+			}
+			if nc.Status.ExternalCIDRNAT == consts.DefaultCIDRValue {
+				remoteExternalCIDRMsg = NotRemappedMsg
+			} else {
+				remoteExternalCIDRMsg = nc.Status.ExternalCIDRNAT
+			}
+
+			// Collect Original Network Configs
+			originalSection := selectedSection.AddSection("Original NetworkConfig")
+			originalSection.AddEntry("Pod CIDR", nc.Spec.PodCIDR)
+			originalSection.AddEntry("External CIDR", nc.Spec.ExternalCIDR)
+
+			// Collect Remapped Network Configs
+			remoteSection := selectedSection.AddSectionWithDetail("Remapped NetworkConfig", remoteSectionMsg)
+			remoteSection.AddEntry("Pod CIDR", remotePodCIDRMsg)
+			remoteSection.AddEntry("External CIDR", remoteExternalCIDRMsg)
+		}
+	}
+
+	pic.addVpnSection(networkSection, tunnelEndpoint)
+	return nil
+}
+
+// addVpnSection adds a section about the VPN configuration.
+func (pic *PeerInfoChecker) addVpnSection(rootSection output.Section, tunnelEndpoint *netv1alpha1.TunnelEndpoint) {
+	tunnelEndpointSection := rootSection.AddSection("VPN Connection")
+	vpnEndpointSection := tunnelEndpointSection.AddSection("Gateway")
+	vpnEndpointSection.AddEntry("Local", tunnelEndpoint.Status.GatewayIP)
```

Comment:
Te IP shown here is that of the gateway pod, not that announced to the other cluster (which might that of the service, or remapped). You can find the correct one in the network config.